libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -203,14 +217,17 @@ void stability_test()
                     motors.set_throttle(throttle_in);
                     motors.set_desired_spool_state(AP_Motors::DesiredSpoolState::THROTTLE_UNLIMITED);
                     update_motors();
+                    SRV_Channels::calc_pwm();
```

Comment:
Does this provide the same output as the heli code. I realize it uses the motors update method but heli does not use the same methods to control the servos.

Reply:
We need this line for the tail servo. The swash outputs direct PWM but the tail sets a scaled value so we need to call `calc_pwm` to convert that to a PWM value that we then record via `hal.rcout->read(i)`